File: Server/Client/src/app/models/bindings/turn-detective-event.ts
import { TicketBag } from "../ticket-bag";
import { ChoosableRoute } from "./choosable-route";
import { IngameEvent } from "./ingame-event";

export interface TurnDetectiveEvent extends IngameEvent {
  villianTickets: TicketBag;
  routes: ChoosableRoute[];
}

File: Server/Application/Hubs/Models/TurnDetectiveEvent.cs
using Server.Application.Hubs.Models;
using Server.Game.Models.Match;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Application.Hubs.Models
{
    public class TurnDetectiveEvent : IngameEvent
    {
        public TicketBag VillianTickets { get; set; }
        public List<ChoosableRoute> Routes { get; set; }
    }
}
